nixos/modules/misc/ids.nix

```diff
@@ -346,6 +346,7 @@ in
       zoneminder = 314;
       paperless = 315;
       #mailman = 316;  # removed 2019-08-30
+      burp = 317;
```

Comment:
Can we use dynamic users?

Reply:
I'm not familiar with this feature. Do you have a link I could read up on? Thanks!

Reply:
IIRC dynamic users potentially requires a chown run on start of the service, which is not practical on a big data store with potentially millions of files, so I don't think using dynamic users feasable for burp.